src/Startup/README.md

```diff
@@ -23,7 +23,24 @@ Unravel provides a fully configurable ASP.NET Core `IWebHost` on top of `System.
 
 ## Setup
 
-1. Install `Unravel.Startup`
+1. Install `Unravel.Startup` in your Web Application project
+1. Search the project for an existing [OWIN Startup Class](https://learn.microsoft.com/en-us/aspnet/aspnet/overview/owin-and-katana/owin-startup-class-detection):
+    - There may be an `OwinStartup` attribute, e.g.
+
+      ```csharp
+      [assembly: OwinStartup(typeof(UnravelExamples.Web.OwinStartup))]
+      ```
+
+    - `Web.config` may contain `owin:appStartup`, e.g.
+
+      ```xml
+      <appSettings>
+        <add key="owin:appStartup" value="UnravelExamples.Web.OwinStartup" />
+      </appSettings>
+      ```
+
+    In the instructions that follow, use that class name instead of `Startup` (or rename it to `Startup`).
```

Comment:
Looking at this again... When there is such a class, it's hard to know what to change about the subsequent steps (3 and 4). _Maybe_ this line is just referring to step 4, which case maybe `instructions that follow` --> `step 4 below`

I know it doesn't _really_ matter. And I acknowledge/suspect the point is to move the project in the direction of having a Startup.cs (as seems to be the norm in newer .NET projects). Motivation is just to add clarity for a newcomer from the get-go.

Reply:
In most cases I expect it's literally just replace `Startup` with your class name:
- Rename the `Global.asax.cs` class to `OwinStartup` instead of `Startup`
- In `OwinStartup.cs`, make the class `partial` and replace `void Configuration(IAppBuilder app)` with `override void ConfigureOwin(IAppBuilder app)`

You're absolutely correct that renaming to `Startup.cs` is probably the more conventional path. _Or_ you could even leave `OwinStartup.cs` as a separate file with `partial class Startup` and `ConfigureOwin()`, leaving a _separate_ `Startup.cs` to have `ConfigureServices()` and eventually Core `Configure(IApplicationBuilder app)`.

Maybe we should replace this line with "See **Startup Discovery** section below." and write up more concretely how this all works?